io.github.OpenToonz.yaml

```diff
@@ -0,0 +1,174 @@
+app-id: io.github.OpenToonz
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.10
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.gfortran-62
+build-options:
+  append-path: /usr/lib/sdk/gfortran-62/bin
+command: opentoonz
+finish-args:
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+modules:     
+  - name: gfortran
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/gfortran-62/install.sh
+
+  - name: glu
```

Comment:
should use shared-module https://github.com/flathub/shared-modules/

Reply:
Oh, I can see the issue here of json vs yaml. Interesting.

Reply:
Actually, I tried shared-module with another app, but I think the added complexity is not worth it.

Reply:
What is exactly the problem and what complexity are we talking about?

Reply:
Aside from this discussion, lgtm